### PR TITLE
Enable ApplePay by adding merchand ID

### PR DESCRIPTION
**Pull request by paul-daniel-dempsey**
## What does this change?
Apple needs to authorise the guardian to display an Apple Pay logo button inside our Epic's on `http://www.theguardian.com/` and `https://m.code.dev-theguardian.com` - a MerchantId file has been authorised and generated for use.

Two (different pre-generated static) Apple merchantId files need to be located at the following paths:-
`https://m.code.dev-theguardian.com/.well-known/apple-developer-merchantid-domain-association`
`https://www.theguardian.com/.well-known/apple-developer-merchantid-domain-association`

This is part of the “one-click payment” exploration, we decided to build in the apple logo button and direct users to checkout to monitor click through rates (only).
[**Trello**](https://trello.com/c/EFtKKTMB/1589-adding-apple-pay-logo-on-to-epics-development)

## Associated PRs
[**Platform**](https://github.com/guardian/platform/pull/1554)
[**Dotcom-Components**](https://github.com/guardian/support-dotcom-components/pull/988)

## Testing Locally (Facia)
http://localhost:9000/.well-known/apple-developer-merchantid-domain-association

## Checklist
- [x] Tested locally, and on CODE if necessary
- [x] Will not break dotcom-rendering

### CODE REVIEW

**Review by mxdvl — COMMENTED** [review]
Could you make sure the `routes` file’s diff is reduced to only the relevant parts?

File: facia/conf/application.conf [hunk 1]
```diff
@@ -6,6 +6,9 @@
 "assets.cache./public/security.txt"="public, max-age=900"
 "assets.cache./public/security.txt.asc"="public, max-age=900"
 
+# Caching for applepay merchantId file
+"assets.cache./public/apple-developer-merchantid-domain-association-code.txt"="public, max-age=900"
```
Comment (mxdvl) [inline facia/conf/application.conf]: I’m not entirely sure how the `max-age` property will be interpreted by our CDN. Is there any risk if it’s cached for longer than 15 minutes?

**Review by mxdvl — APPROVED** [review]
LGTM – you probably want to squash your commits!

Have you checked that it works locally?

File: facia/conf/application.conf (hunk 1 above)
Comment (jamesgorrie) [reply facia/conf/application.conf]: Max age is respected from app => router => fastly.
I think on average `frontend` controls most the cache times.

**Review by jamesgorrie — APPROVED** [review]
Fantastic

**Comment by jamesgorrie** [comment]
As we need to serve assets conditionally, we've had to make the `Assets` controller available to the `FaciaController`.

It is odd that we've not seen this before, but a lot of the work to do that is wiring up the dependencies.

It does point to that we should probably not be relying on the facia controller to serve static text files. Maybe Fastly can help us here/

**Pull request merged**